server.js

```diff
@@ -73,7 +73,7 @@ app.prepare().then(() => {
     } else if (/^\/doc\/tutorial\/(.*)?/.test(pathname)) {
       // path /doc/tutorial/... -> /doc/tutorials/deep/...
       res.writeHead(301, {
-        Location: req.url.replace('/doc/tutorial/', '/doc/tutorials/deep/')
+        Location: req.url.replace('/doc/tutorial/', '/doc/tutorials/deep')
```

Comment:
It does not work properly for the case like `/doc/tutorial/something`

Reply:
Yeah there was no reason to change this, I was confused. I reverted this, thanks.
But I updated the other one that matched exact path strings `/doc/tutorial` or `/doc/tutorial/` to match regex `/^\/doc\/tutorial\/*$/i` now (redirects to /doc/tutorials).